zstash/extract.py

```diff
@@ -365,16 +369,23 @@ def extractFiles(  # noqa: C901
 
         try:
             # Seek file position
-            # FIXME: Item "None" of "Optional[IO[bytes]]" has no attribute "seek" mypy(error)
-            tar.fileobj.seek(file_tuple[6])  # type: ignore
+            if tar.fileobj:
+                fileobj = tar.fileobj
+            else:
+                raise Exception("Invalid tar.fileobj={}".format(tar.fileobj))
+            fileobj.seek(file_tuple[6])
 
             # Get next member
             tarinfo = tar.tarinfo.fromtarfile(tar)
 
             if tarinfo.isfile():
                 # fileobj to extract
+                extracted_file = tar.extractfile(tarinfo)
+                if extracted_file:
+                    fin = extracted_file
+                else:
+                    raise Exception("Invalid extracted_file={}".format(extracted_file))
```

Comment:
You should use raise ValueError(...) to be more explicit on what the exception is.

https://docs.python.org/3/library/exceptions.html#ValueError
https://stackoverflow.com/a/43605619